src/logging/mod.rs

```diff
@@ -59,6 +60,46 @@ impl KV for RequestMozlogFields {
     }
 }
 
+#[derive(Clone)]
+struct AppErrorFields {
+    code: String,
```

Comment:
Doesn't really matter tbh, it's only logging, but this one could/should be an `i32` as well, right?

Reply:
(actually `u16` is probably more correct for both if I'm feeling pedantic, which I usually am)